helm/templates/route.yaml

```diff
@@ -12,10 +12,11 @@ metadata:
     haproxy.router.openshift.io/timeout: {{ .Values.route.timeout.duration }}
     {{- end }}
 spec:
-  path:  {{ .Values.route.path | default "/" }}
+  path: {{ .Values.route.path | default "/" }}
   to:
     kind: Service
     name: {{ $releaseName }}-{{ $chartName }}
   tls:
     termination: edge
+    insecureEdgeTerminationPolicy: Redirect
```

Comment:
Seems like certs for the route are missing here: 
see ref:
https://github.com/MapColonies/geoserver-api/blob/master/helm/templates/route.yaml